File: 1464-maximum-product-of-two-elements-in-an-array/1464-maximum-product-of-two-elements-in-an-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var maxProduct = function(nums) {
    let mul = -1;
    for(let i=0; i<nums.length; i++){
        for(let j=0; j<nums.length; j++){
            if(i !== j){
                if((nums[i]-1)*(nums[j]-1) > mul){
                    mul = (nums[i]-1)*(nums[j]-1);
                }
            }
        }
    }
    return mul;
};

File: 1464-maximum-product-of-two-elements-in-an-array/1464-maximum-product-of-two-elements-in-an-array.java
class Solution {
    public int maxProduct(int[] nums) {
        int max = 0;
        for(int i=0; i<nums.length; i++){
            int temp = 0;
            for(int j=0; j<nums.length; j++){
                if(i != j){
                    temp = (nums[i] - 1) * (nums[j] - 1);
                }
                if(max < temp){
                max = temp;
                }
            }
        }
        return max;
    }
}